ci-operator/step-registry/ipi/conf/azuremag/ipi-conf-azuremag-commands.sh

```diff
@@ -4,6 +4,9 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
Refer to https://github.com/openshift/release/pull/31256, we are trying to remove the command of downloading `yq` to avoid network issue.

I think you can update `ipi-conf-azuremag-ref.yaml` to use upi-instealler:4.12 image instead.